src/views/Login.vue

```diff
@@ -48,11 +53,21 @@ export default defineComponent({
   data() {
     return {
       username: "",
-      password: ""
+      password: "",
+      instanceUrl: ""
     };
   },
+  computed: {
+    ...mapGetters({
+      currentInstanceUrlSaved: 'user/getInstanceUrl'
```

Comment:
```suggestion
      currentInstanceUrl: 'user/getInstanceUrl'
```

Reply:
Changed the name.
Thank You Sir for reviewing my code.